config/initializers/govuk_publishing_components.rb

```diff
@@ -0,0 +1,12 @@
+if defined?(GovukPublishingComponents)
+  GovukPublishingComponents.configure do |c|
+    c.component_guide_title = "Static Component Guide"
+    c.static = true
+
+    # Component guide already includes static, which includes the components
```

Comment:
This sounds pretty circular?

Reply:
Aye, that's the problem this PR has to solve/work around. As @h-lame nicely put it, the shape of the world we've built isn't quite right.